src/types/errors.ts

```diff
@@ -0,0 +1,6 @@
+export interface CozyBackendError {
+  status: number
+  title: string
+  detail: string
+  source: { parameter: string }
+}
```

Comment:
This type should be defined in CozyClient, not here.

Reply:
Definitely as well as the Alerter methods. But in this context (prod monday) could we make an exception? I'll update the types later for cozy-client and cozy-ui

Reply:
The PR has been approved. So let's move on ;).

You can create an issue on cozy-client's side for the TypeError. 

For the Alerter, you can discuss about that here https://github.com/cozy/cozy-client/issues/552